src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -1598,4 +1598,13 @@ This version used in: {2}</value>
   <data name="Use_expression_body_for_local_functions" xml:space="preserve">
     <value>Use expression body for local functions</value>
   </data>
+  <data name="Unwrap calls" xml:space="preserve">
+    <value>Unwrap calls</value>
+  </data>
+  <data name="Wrap_calls" xml:space="preserve">
+    <value>Wrap calls</value>
+  </data>
+  <data name="Wrap_long_calls" xml:space="preserve">
+    <value>Wrap long calls</value>
```

Comment:
Alternative names:

1. Unwrap call chain
2. Wrap call chain
3. Wrap long call chain

Reply:
definitely `Wrap call chain`

long lines doesn't communicate that this is operating on, well, call chains.

Reply:
I didn't noticed there is two options, how about

- Wrap call chains
- Wrap call chains on separate lines

"long" there was a little confusing to me.

Reply:
both of those would be wrapping to separate lines.  So it's unclear to me how those two actually differ.

We use the 'long' term already for the existing wrapping functions.  Specifically "wrap long parameter list" "wrap long argument list" and "wrap long expression".  It basically means "wrap once you go past 120 chars".